jp/components/grids_and_lists.md

```diff
@@ -20,176 +20,176 @@ _keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Nati
+      <a href="grid_virtualization.md" class="feature__more-details">行仮想化の実装</a>
     </div>
     <div class="feature__details">
-      <h3 id="virtual-row">[Virtual Row](grid_virtualization.md)</h3>
-      <p>Quickly navigate through on-screen rows with the rest rendering immediately upon scrolling.</p>
+      <h3 id="virtual-row">[行の仮想化](grid_virtualization.md)</h3>
+      <p>スクロールに合わせて直ちに描画される行をすばやくナビゲーションします。</p>
     </div>
   </div>
 
   <div class="feature">
     <div class="feature__image feature__image--right">
-      <img class="b-lazy" src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png" data-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-768.jpg" data-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-480.jpg 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-768.jpg  768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-1100.jpg 1100w" sizes="(min-width: 1800px) 575px,(max-width: 1799px) and (min-width: 1200px) 50vw,(max-width: 1199px) and (min-width: 992px) 33vw,(max-width: 991px) and (min-width: 768px) 50vw, 100vw" alt="Virtual column rendering for scrolling in Angular Data Grid.">
-      <a href="grid_virtualization.md" class="feature__more-details">Virual Column Implementation</a>
+      <img class="b-lazy" src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png" data-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-768.jpg" data-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-480.jpg 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-768.jpg  768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-cell-virtual-scrolling-1100.jpg 1100w" sizes="(min-width: 1800px) 575px,(max-width: 1799px) and (min-width: 1200px) 50vw,(max-width: 1199px) and (min-width: 992px) 33vw,(max-width: 991px) and (min-width: 768px) 50vw, 100vw" alt="Angular Data Grid のスクロール時の列描画の仮想化">
+      <a href="grid_virtualization.md" class="feature__more-details">列仮想化の実装</a>
     </div>
     <div class="feature__details">
```

Comment:
I get the feeling that this is a duplicated section.  I'd recommend that it be fixed in English, but we should remove it from Japanese regardless.